### PR TITLE
completed spatial aggregation examples in ch 03

### PR DESCRIPTION
**Pull request by michaeldorman**

### CODE REVIEW

**Comment by michaeldorman** [comment]
@Robinlovelace @Nowosad @anitagraser Could you please check and approve or suggest changes? Thanks 🙏

**Comment by Nowosad** [comment]
Quarto syntax looks good. (I am not going to comment on the Python code here)

File: 02-spatial-data.qmd [hunk 1]
```diff
@@ -2,7 +2,7 @@
 jupyter: python3
 ---
 
-# Geographic data in Python {#spatial-class}
+# Geographic data in Python {#sec-spatial-class}
```
Comment (Robinlovelace) [inline 02-spatial-data.qmd:5]: :+1: to consistent and clear refs.

**Review by Robinlovelace — APPROVED** [review]

**Comment by michaeldorman** [comment]
Thanks @Nowosad and @Robinlovelace !

File: 03-attribute-operations.qmd [hunk 2]
```diff
@@ -55,7 +55,7 @@ src_multi_rast = rasterio.open('data/landsat.tif')
 
 Attribute data is non-spatial information associated with geographic (geometry) data.
 A bus stop provides a simple example: its position would typically be represented by latitude and longitude coordinates (geometry data), in addition to its name.
-The Elephant & Castle / New Kent Road stop in London, for example has coordinates of `-0.098` degrees longitude and `51.495` degrees latitude which can be represented as `POINT (-0.098 51.495)` in the Simple Feature representation described in Chapter @spatial-class.
+The Elephant & Castle / New Kent Road stop in London, for example has coordinates of `-0.098` degrees longitude and `51.495` degrees latitude which can be represented as `POINT (-0.098 51.495)` in the Simple Feature representation described in @sec-spatial-class.
```
[the remark this answers is not in the document]
Comment (Nowosad) [reply 03-attribute-operations.qmd:58]: It should work. This is a quarto syntax for references (you do not need to add "Chapter"/"Section"; it is done automatically).

**Pull request merged**